src/disco/containers/InstallButton.js

```diff
@@ -71,39 +117,9 @@ export class InstallButton extends React.Component {
 }
 
 export function mapStateToProps(state, ownProps) {
-  return (state.installations || {})[ownProps.slug] || {};
-}
-
-export function mapDispatchToProps(dispatch, ownProps) {
-  const { slug } = ownProps;
-  const url = 'foo';
-  const guid = 'foo@foo.com';
-  dispatch({type: 'INSTALL_STATE', payload: {slug, guid, url, status: UNINSTALLED}});
-  /* istanbul ignore next */
-  return {
-    install() {
-      dispatch({type: 'START_DOWNLOAD', payload: {slug, guid, url}});
-      let downloadProgress = 0;
-      const downloading = setInterval(() => {
-        downloadProgress += Math.ceil(Math.random() * 10 + 15);
-        downloadProgress = Math.min(downloadProgress, 100);
-        dispatch({type: 'DOWNLOAD_PROGRESS', payload: {slug, downloadProgress}});
-        if (downloadProgress >= 100) {
-          clearInterval(downloading);
-          dispatch({type: 'START_INSTALL', payload: {slug}});
-          setTimeout(() => {
-            dispatch({type: 'INSTALL_COMPLETE', payload: {slug}});
-          }, 5000);
-        }
-      }, 500);
-    },
-    uninstall() {
-      dispatch({type: 'START_UNINSTALL', payload: {slug}});
-      setTimeout(() => {
-        dispatch({type: 'UNINSTALL_COMPLETE', payload: {slug}});
-      }, 5000);
-    },
-  };
+  const installation = (state.installations || {})[ownProps.slug] || {};
```

Comment:
There's a test that checks that it's there, but yes I could remove it along with the test.